PRINCIPLES.md

```diff
@@ -10,22 +10,22 @@ These principles were derived from modern software operations but are rooted in
 
 ## Principles
 
-1. **The principle of declarative desired state**
+1. **Desired System State Is Declarative**
 
     A system managed by GitOps must have its _Desired State_ expressed declaratively as data in a format writable and readable by both humans and machines.
 
-2. **The principle of immutable desired state versions**
+2. **Declarations Are Versioned and Immutable**
 
     _Desired State_ is stored in a way that supports versioning, immutability of versions, and retains a complete version history.
 
-3. **The principle of continuous state reconciliation**
+3. **State Reconciliation Is Continuous**
 
-    Software agents continuously, and automatically, compare a system's _Actual State_ to its _Desired State_.
-    If the actual and desired states differ for any reason, automated actions to reconcile them are initiated.
+    Software agents continuously, and automatically, compare a system's state to its _Desired State_.
+    If these states differ for any reason, automated actions to reconcile them are initiated.
 
-4. **The principle of operations through declaration**
+4. **Intentional Changes Only Through Versioning**
```

Comment:
What about `4. Actualizations to Declarations are versioned`

Reply:
Note: Not sure if these are interchangeable in this context `Actualizations to Declarations...` vs `Updates to Declarations...` vs `Changes to Declarations...`

Reply:
🤔